anti-corruption-layer/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.main.allow-circular-references=true
```

Comment:
Hmm, looks smelly. Any way to get rid of circular references?

Reply:
yeah, I added two-way links and apparently, we can keep only one way when we check the existence of the order in the old shop from the modern one.